docs/src/content/docs/configuration/core-configuration.md

```diff
@@ -0,0 +1,26 @@
+---
+title: Core Configuration
+description: Summary for core configuration values
+---
+
+## PublicChatTrigger
+
+List of characters to use for public chat triggers.
+
+## SilentChatTrigger
+
+List of characters to use for silent chat triggers.
+
+## FollowCS2ServerGuidelines
+
+Per [CS2 Server Guidelines](https://blog.counter-strike.net/index.php/server_guidelines/), certain plugin
+functionality will trigger all of the game server owner's Game Server Login Tokens
+(GSLTs) to get banned when executed on a Counter-Strike 2 game server.
+
+Enabling this option will block plugins from using functionality that is known to cause this.
+This option only has any effect on CS2. Note that this does NOT guarantee that you cannot
+receive a ban.
```

Comment:
I think this comment is redundant (probably a copy pasta)
```suggestion
Note that this does NOT guarantee that you cannot
receive a ban.
```

Reply:
Committed suggestion, yes it is indeed a reference from SM